ipynb/sched_dvfs/smoke_test.ipynb

```diff
@@ -324,7 +324,7 @@
     "    rtapp.run(out_dir=exp_dir)\n",
     "    \n",
     "    # Stop FTrace and sample Energy consumption\n",
-    "    nrg = te.emeter.sample()\n",
+    "    nrg = te.emeter.report(exp_dir)[0]\n",
```

Comment:
You was right, returning a tuple is at least horrible :-/
Should we better think about some form of named tuple?

```python
EnergyReport = namedtuple ('EnergyReport', ['channels', 'report_file'])
```

which should then allow to consume the data with something like
```python
er = te.emeter.report(exp_dir)
nrg = er.channels
```